docs/pages/build-reference/variables.md

```diff
@@ -159,3 +159,4 @@ The following environment variables are exposed to each build job:
 - `EAS_BUILD_PROFILE` - the name of the build profile from `eas.json`, e.g. `release`
 - `EAS_BUILD_GIT_COMMIT_HASH` - the hash of the Git commit, e.g. `88f28ab5ea39108ade978de2d0d1adeedf0ece76`
 - `EAS_BUILD_NPM_CACHE_URL` - the URL of the npm cache ([learn more](how-tos.md#using-npm-cache-with-yarn-v1))
+- `EAS_BUILD_USERNAME` - the username of the user initiating the build (it's undefined for bot users)
```

Comment:
hm, i think i misunderstood the meaning of this field. i had thought that this would map to the username of the owner, eg: if i am notbrent, belonging to the easbuild org, and i kick off a build for an app with `owner: 'easbuild'`, then i would expect `EAS_BUILD_USERNAME` to be easbuild.

Reply:
I'll merge this for now. I'll make sure to get on the same page with you why you'd like it to be set to the account name.